cppcache/integration-test/CMakeLists.txt

```diff
@@ -69,6 +67,10 @@ add_clangformat(unit_test_callbacks)
 
 enable_testing()
 
+if(UNIX AND NOT APPLE)
```

Comment:
This seems a little wonky... Is this perhaps a Linux or GCC specific dependency? It is also likely that our dependency or `rt` comes from another library and if so that is where this dependency should be added. Also, see root CMakeLists.txt for some of these more platform specific dependencies.

Reply:
Perhaps something more like this?
```
find_library(LIB_RT  rt)
  if (LIBRT)
    target_link_libraries(<target> ${LIB_RT})
  endif()
```

Reply:
It's true that my solution for this is not the best one.
Regarding root CMakeLists.txt I don't think it makes sense, as rt is only needed for the shared memory API, which is used only by integration-tests. I'll try the find_library approach for this

Reply:
Oh, well then if it is just these legacy tests... hack away.